Source/WebKit/NetworkProcess/cache/NetworkCacheStorage.h

```diff
@@ -198,8 +201,10 @@ class Storage : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Storage,
     HashSet<std::unique_ptr<ReadOperation>> m_activeReadOperations;
     WebCore::Timer m_readOperationTimeoutTimer;
 
+    Lock m_activitiesLock;
+    HashCountedSet<uint64_t, AlreadyHashed> m_writeOperationActivities WTF_GUARDED_BY_LOCK(m_activitiesLock);
     Deque<std::unique_ptr<WriteOperation>> m_pendingWriteOperations;
-    HashSet<std::unique_ptr<WriteOperation>> m_activeWriteOperations;
+    HashMap<uint64_t, std::unique_ptr<WriteOperation>> m_activeWriteOperations;
```

Comment:
I'm curious. Don't we prefer to use ObjectIdentifiers for identifiers? Additionally since we are using the identifier as a key to the HashMap and don't want to use a value that would corrupt the HashMap.

Reply:
I think we do if the identifier is used in different classes and especially across processes (e.g. included in IPC message). In this case the identifier is only used internally in `NetworkCache::Storage`.

Reply:
I see. Wouldn't it be better to use ObjectIdentifier here as well for the sake of consistency?

Reply:
What consistency? I mean the identifier is used only inside `NetworkCache::Storage` so it does not need to be `ObjectIdentifier`

Reply:
Oh I just meant consistency with other ObjectIdentifiers.

Reply:
> I think we do if the identifier is used in different classes and especially across processes

Our C++ guidelines certainly do not make any such distinction. Any identifier should be an ObjectIdentifier.

Reply:
ok I was still under the impression that is only needed for IPC code and saw ReadOperation identifier in the same file uses uint64_t, but it seems the latest guideline is for all code.
Updated.